transport/wseb/src/test/java/org/kaazing/gateway/transport/wseb/specification/wse/connector/OpeningIT.java

```diff
@@ -335,33 +310,6 @@ public void shouldFailConnectionWhenXWebSocketProtocolNotNegotiated() throws Exc
     }
 
     @Test
```

Comment:
We should really still test this as it is part of the spec, but to do this you would need to mock the extension factory or something...